JfkWebApiSkills/JfkInitializer/Program.cs

```diff
@@ -45,10 +49,11 @@ static void Main(string[] args)
             SynonymMapName = ConfigurationManager.AppSettings["SynonymMapName"];
             BlobContainerNameForImageStore = ConfigurationManager.AppSettings["BlobContainerNameForImageStore"];
 
-            string searchServiceName = ConfigurationManager.AppSettings["SearchServiceName"];
+            Uri searchServiceEndpoint = new Uri(string.Format("https://{0}.{1}", ConfigurationManager.AppSettings["SearchServiceName"], ConfigurationManager.AppSettings["SearchServiceDnsSuffix"]));
```

Comment:
Use `UriBuilder` instead.

Reply:
Looking at it, I don't think using UriBuilder helps us get rid of the string.Format here (which I assume is what you were going for) because what we are building is the whole Host property anyways. So I'm just going to leave it as is for simplicity.